src/silx/io/commonh5.py

```diff
@@ -206,6 +206,11 @@ def _is_editable(self):
         f = self.file
         return f is None or f.mode == "w"
 
+    def __hash__(self):
+        if self.file is None:
+            return hash((self.__class__, None, self.name))
```

Comment:
I would say the None is pointless here, no?

Reply:
This is being reworked in #3665.
Yes it was to keep the pattern.... but it is useless